src/types/commands.ts

```diff
@@ -421,7 +425,7 @@ export interface MoveSheetCommand extends SheetDependentCommand {
   delta: number;
 }
 
-export interface RenameSheetCommand extends SheetDependentCommand {
+export interface RenameSheetCommand extends SheetDependentCommand, SheetEditingCommand {
   type: "RENAME_SHEET";
```

Comment:
for RENAME_SHEET, it would be best to have `newName` and `oldName` or something like that. Because it's impossible to know what is what with `name` and `sheetName`.